frontend/app/templates/components/inplace_editor/editable/textarea.html

```diff
@@ -2,7 +2,8 @@
           class="focus-input inplace-edit--textarea"
           name="value"
           ng-disabled="fieldController.state.isBusy"
+          ng-required="editPaneController.isRequired"
           ng-model="fieldController.writeValue.raw"
-          title="{{ editTitle }}"
+          title="{{ fieldController.editTitle }}"
```

Comment:
You were talking about merge-conflicts: Not sure if this is one, that got resolved badly...

Could also be an intended change, I am unsure...